res/controllers/common-controller-scripts.js

```diff
@@ -326,9 +326,8 @@ script.midiPitch = function (LSB, MSB, status) {
    -------- ------------------------------------------------------ */
 script.spinback = function(channel, control, value, status, group, factor, rate) {
     // if brake is called without defined factor and rate, reset to defaults
-    if (factor === undefined && rate === undefined) {
```

Comment:
Why did you remove this?

It would be better to reimplement the `rate` default with a separate statement though. In the event that `factor` was defined but `rate` was not, both parameters would be set to the defaults.

Reply:
ping

Reply:
I changed this because it factor was undefined but rate was (which I think can happen in javascript if the variable passed in is undefined?), it would be passed through to spinback without being set.  Mostly it's a noop to remove code duplication.  Rate is checked in the existing code, below.

Reply:
Ah sorry, GitHub hid the code below so I missed that.